Components/Combat/DefaultTargetingProvider.cs

```diff
@@ -244,38 +243,37 @@ public bool IsInRange(Vector3 position, TrinityPower power)
             var rangeRequired = Math.Max(1f, power.MinimumRange);
             var distance = position.Distance(Core.Player.Position);
 
-            TrinityActor currentTarget = TrinityCombat.Targeting.CurrentTarget;
-            if (Core.Player.IsInBossEncounter && currentTarget != null)
+            if (Core.Player.IsInBossEncounter && TrinityCombat.Targeting.CurrentTarget != null)
             {
-                var positionIsBoss = currentTarget.IsBoss && currentTarget.Position.Distance(position) < 10f;
+                var positionIsBoss = TrinityCombat.Targeting.CurrentTarget.IsBoss && TrinityCombat.Targeting.CurrentTarget.Position.Distance(position) < 10f;
                 if (positionIsBoss)
                 {
-                    rangeRequired += currentTarget.CollisionRadius;
+                    rangeRequired += TrinityCombat.Targeting.CurrentTarget.CollisionRadius;
                 }
             }
 
             Core.Logger.Verbose(LogCategory.Targetting, $">> CurrentPower={power} CurrentTarget={position} RangeReq:{rangeRequired} Dist:{distance}");
 
-            // Handle Belial differently, he's never in LineOfSight.
-            if (Core.Player.IsInBossEncounter && currentTarget != null && currentTarget.ActorSnoId == (int) SNOActor.Belial)
+            // Handle Belial differently, he's never in LineOfSight. Replace CurrentTarget to CurrentTarget for fix error
+            if (Core.Player.IsInBossEncounter && CurrentTarget != null && CurrentTarget.ActorSnoId == (int) SNOActor.Belial)
                 return distance <= rangeRequired;
 
             return distance <= rangeRequired && IsInLineOfSight(position);
         }
 
-        public bool IsInLineOfSight(TrinityActor currentTarget)
+        public bool IsInLineOfSight(TrinityActor CurrentTarget)

```

Comment:
Please revert this change (MS naming convention uses camelCase for parameter names).